src/pymovements/gaze/screen.py

```diff
@@ -20,20 +20,21 @@
 """Provides the Screen class."""
 from __future__ import annotations
 
+from dataclasses import dataclass
+
 import numpy as np
 
 from pymovements.gaze import transforms_numpy
 from pymovements.utils import checks
-from pymovements.utils import decorators
 
 
-@decorators.auto_str
+@dataclass
 class Screen:
     """Screen class for holding screen properties.
 
-     Also transforms pixel coordinates to degrees of visual angle.
+    Also transforms pixel coordinates to degrees of visual angle.
```

Comment:
this comment might be confusing if people want to use pixel coordinates instead of dva.

Reply:
I removed the line completely from the docstring to reduce some of the confusion.

The thing with the `pix2deg()` inside this class is a tough knot to unfold.

It relies on our legacy numpy functions that still aren't officially deprecated.
The `y_max_dva` etc. properties rely on this. The heatmap plot relies on the `y_max_dva` properties.

I think the functionality itself shouldn't be removed, but maybe it could fit somewhere else better? I don't know.